.github/workflows/testing.yml

```diff
@@ -9,38 +9,20 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16]
```

Comment:
suggestion: Node.js 16 is going to be deprecated on 2023-09-11 I suggest to switch tests to run on LTS version which is 18.

Reply:
I think it's risky to not test on the currently popular version. In fact, I've intentionally targeted tsc to Node 12,so maybe expand the build matrix to include 12, 14, 16, and 18?

Reply:
I agree lets expand the matrix as you suggest.